docs/development_suite/overview-dev-suite.md

```diff
@@ -25,200 +25,11 @@ The tool to reach this goal is the **Internal Developer Portal** which will help
 
 Mia-Platform Console allows you to govern all projects in one place, industrialize & automate DevOps, avoid organizational bottlenecks, self-serve developers, solve cloud complexity and enhance accountability & reliability.
 
-## Homepage
+**Hierarchical Organization of Mia-Platform Console:**
 
-There are two different Homepages:
+- **Console:** The overarching workspace and management environment, serving as the primary gateway to the platform.
+- **Company:** An organizational entity that acts as a container for projects. It can represent the entire organization or a specific operational unit.
+- **Project:** A structured collection of microservices, configurations, and data. It might represent a full digital system or a specific component.
+- **Project Sections:** Designated areas within a project that delineate and organize functionalities and tasks.
 
-* [The Basic Homepage](#basic-homepage) allows you to view the various features offered by the Console and navigate them easily by clicking on one of the cards.
-* [The Homepage Metrics](#metrics-homepage) allows you to view the status of the project, including environments, pods status, and requests / limits of CPU and RAM.
-  
-:::info
-You can switch between the Homepages using the `⌘ + K` (alternatively, `ctrl + K`) keyboard shortcut.
-:::
-
-### Basic Homepage
```

Comment:
these changes are already present in main and [out in production](https://docs.mia-platform.eu/docs/development_suite/overview-dev-suite) I think the PR needs rebasing for this change to disappear